extensions/mssql/src/objectManagement/localizedConstants.ts

```diff
@@ -277,6 +281,20 @@ export const NumberOfUsersText = localize('objectManagement.databaseProperties.n
 export const MemoryAllocatedText = localize('objectManagement.databaseProperties.memoryAllocated', "Memory Allocated To Memory Optimized Objects");
 export const MemoryUsedText = localize('objectManagement.databaseProperties.memoryUsed', "Memory Used By Memory Optimized Objects");
 export const StringValueInMB = (value: string) => localize('objectManagement.databaseProperties.mbUnitText', "{0} MB", value);
+export const AutoCreateIncrementalStatisticsText = localize('objectManagement.databaseProperties.autoCreateIncrementalStatisticsText', "Auto Create Incremental Statistics");
+export const AutoCreateStatisticsText = localize('objectManagement.databaseProperties.AutoCreateStatisticsText', "Auto Create Statistics");
+export const AutoShrinkText = localize('objectManagement.databaseProperties.autoShrinkText', "Auto Shrink");
+export const AutoUpdateStatisticsText = localize('objectManagement.databaseProperties.autoUpdateStatisticsText', "Auto Update Statistics");
+export const AutoUpdateStatisticsAsynchronouslyText = localize('objectManagement.databaseProperties.autoUpdateStatisticsAsynchronouslyText', "Auto Update Statistics Asynchronously");
+export const IsLedgerDatabaseText = localize('objectManagement.databaseProperties.isLedgerDatabaseText', "Is Ledger Database");
+export const PageVerifyText = localize('objectManagement.databaseProperties.pageVerifyText', "Page Verify");
+export const TargetRecoveryTimeInSecondsText = localize('objectManagement.databaseProperties.targetRecoveryTimeInSecondsText', "Target Recovery Time (Seconds)");
+export const DatabaseReadOnlyText = localize('objectManagement.databaseProperties.databaseReadOnlyText', "Database Read-Only");
+export const DatabaseStateText = localize('objectManagement.databaseProperties.databaseStateText', "Database State");
+export const EncryptionEnabledText = localize('objectManagement.databaseProperties.encryptionEnabledText', "Encryption Enabled");
+export const UserAccessText = localize('objectManagement.databaseProperties.userAccessText', "Restrict Access");
+export const TrueText = localize('objectManagement.databaseProperties.trueText', "true");
```

Comment:
Do we also need False along with True here?

Reply:
Since im comparing the boolean string value to get the boolean value I just used the true string, I can modify the using typeCasting and remove this one actually!